docs/pages/blog/mui-x-v8.md

```diff
@@ -0,0 +1,380 @@
+
+<figure>
+  <video autoplay muted loop playsinline width="1046" height="720" controls>
+    <source src="/static/blog/mui-x-v8/pivoting.mp4" type="video/mp4">
+  </video>
+  <figcaption>A common pivoting use case</figcaption>
+</figure>
+
+Learn how to use pivoting [now](/x/react-data-grid/pivoting/).
+
+### Ask Your Table - Featuring AI Assistance on the Data Grid [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
+
+After a period of test with users and pilot customers, we're excited to introduce our new AI Assistant for the Data Grid: [Ask Your Table](/x/react-data-grid/ai-assistant/).
+
+Now your users can interact with data using natural language queries and extract data insights quickly as ever.
+
+For example, if you’re working with an Olympics medal dataset, you can simply ask, “Who won the most gold medals this century?” and the Data Grid will automatically apply the right filters, groupings, and aggregations to display your answer—all that while keeping your dataset private.
+
+<figure>
+  <img width="2084" height="928" alt="An example use case showcasing the answer to the question with the correct sorting, filters and grouping" src="/static/blog/mui-x-v8/ask-your-table.png" loading="lazy">
+  <figcaption>An answer from the Data Grid</figcaption>
+</figure>
+
+**Feature's key Benefits:**
+
+- **Enhanced usability**: Simplify interactions with data by asking questions in plain language.
+- **Unlock the full potential of the Data Grid**: Explore, combine, and discover powerful data analysis features often hidden behind menus, without the tedious hopping from one menu to another.
+- **Data privacy:** No need to share your dataset—the service model only requires your column definitions and a few data samples.
+
+Interested in bringing this feature to your users? [Contact us](mailto:sales@mui.com) to get access to your API Key!
+
+### List View [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+We’ve promoted List View to stable, and over this major we’re committed to enhancing this feature even further—making it easier to deliver a robust Data Grid experience on small screens.
+
+<figure>
+  <img width="512" height="768" alt="An example of a Data Grid's list view on a phone" src="/static/blog/mui-x-v8/list-view.png" loading="lazy">
+  <figcaption>An example of a Data Grid's list view on a phone</figcaption>
+</figure>
+
+Learn more about [List View](/x/react-data-grid/list-view/).
+
+### Row Spanning
+
+Row spanning has also been promoted to stable, providing a reliable and flexible solution for merging cells across multiple rows. This enhancement makes it easier to create complex, multi-row layouts within your Data Grid, allowing you to present your data more effectively.
+
+Learn how to setup and use [Row spanning](/x/react-data-grid/row-spanning/).
```

Comment:
"Setup" is a noun vs. "set up" which is a verb. No need to capitalize the R here.

```suggestion
Learn how to set up and use [row spanning](/x/react-data-grid/row-spanning/).
```